### ISSUE TITLE
Empty cells cause error when importing objects or relationship classes from .xlsx or .csv files to .sqlite

### ISSUE THREAD

**Issue by eriharriso** [Spine-project/Spine-Toolbox#1410]
I was trying to import data from an xlsx/csv file which contains empty cells to an sqlite database. Importing relationship classes (linking cells on the same row) causes the following error log in the example project (attached):

```Could not import relationship with objects ('2',) into 'Numbers': Some of the objects in relationship 'Numbers_2' are invalid.```

Trying to import a similar .csv file brings up the following error log:

```
Could not import object '' with class 'Header 2': Object name is an empty string and therefore not valid
Could not import relationship with objects ('2', '') into 'data': Some of the objects in relationship 'data_2__' are invalid.
```

The empty cells are then not imported and neither are any relationship classes from rows that contain empty cells. The same errors appear whether float or string data types are used for the columns.

I have the following Mapping options in the import editor:
Item type: Relationship class
Import objects: ticked
Parameter type: None

Possible solution: the importer importing empty cells as None, null or something similar.

[empty_cell_example.zip](https://github.com/Spine-project/Spine-Toolbox/files/6552357/empty_cell_example.zip)

**Comment by manuelma** [comment]
Have you tried unchecking "cancel on error" in the Importer Properties UI?

**Comment by eriharriso** [comment]
Yes, the errors above resulted with "cancel on error" unchecked. Also, the toolbox does import every relationship (row) in which all cells are non-empty. It just skips the ones in which there is at least one empty cell.

**Comment by manuelma** [comment]
Thanks. It looks like everything is working as intended.

What would be the desired behavior? You don't want to see those errors in the log?

**Comment by eriharriso** [comment]
It seems this error was in the user end: I was trying to import cells as objects where I should have imported them as parameters (which correctly have None as a default value and therefore solve my problem of trying to import a partly incomplete database). Sorry, and thank you for the quick reply!

**Comment by jkiviluo** [comment]
Just a note that this was discussed also here: https://github.com/Spine-project/Spine-Toolbox/issues/1325
